docs/flashbots-protect/blobs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Blobs
+---
+
+The Flashbots Protect RPC accepts blobs. You can send multiple blob transactions in a slot.
+
+### Why use Protect
+
+The public mempool only accepts one blob transaction per user per slot. This means rollups must guess how many blobs to include. It is also expensive to replace blob transactions.
+
+The Flashbots private mempool accepts many blobs per user per slot. Rollups can send many blob transactions and continuously submit new blob transactions throughout the slot. There is no cost to replace blob transactions.
+
+### How to send blobs
+
+Send regular `eth_sendRawTransaction` requests to `rpc.flashbots.net/fast`.
+
+The "fast" option will share your blobs with all block builders for fast inclusion. You can remove it if you only want the Flashbots builder to include your blobs. To receive customer support, include an optional query parameter "originId=your-rollup-name".
+
+### How are blobs merged
+
+Blobs are merged into a block based on their priority fee, the same way that regular transactions and bundles are merged. Builders will ensure no more than 6 blobs are included in a block. If different senders submit more than 6 blobs in total, the combination of blob transactions that pays the most overall will be included.
```

Comment:
Not sure how deep in the weeds we should get here, but it might be worth mentioning that the priority fees only apply to the non-blob gas used by the transaction.  Thus a tx that just sends blobs but doesn't actually use much EVM gas needs to specify a very high priority fee to be included.  For example BlobScriptions only use ~21272 gas.